packages/tooltips/examples/boundary-detection.md

```diff
@@ -0,0 +1,23 @@
+```jsx
```

Comment:
Preface with some explanatory text about the user's need to scroll. Otherwise, it just looks like a broken (empty) example box.

Reply:
I was thinking something similar before seeing this comment. My thought was to include some explanatory text within the example, but explanatory outside the example would be cool too.

```
// Scroll down to view tooltip with collision detection.










<Tooltip>






```

Reply:
Now that we have defaulted the `boundariesElement` to be against the `viewport` this examples doesn't really show much. I'm gong to remove it.